jobs/JENKFGDAS_ARCHIVE

```diff
@@ -0,0 +1,91 @@
+#! /usr/bin/env bash
+
+source "$HOMEgfs/ush/preamble.sh"
+
+#############################
+# Source relevant config files
+#############################
+export EXPDIR=${EXPDIR:-${HOMEgfs}/parm/config}
+configs="base earc"
+config_path=${EXPDIR:-${NWROOT}/gfs.${gfs_ver}/parm/config}
+for config in ${configs}; do
+    . ${config_path}/config.${config}
+    status=$?
+    [[ ${status} -ne 0 ]] && exit ${status}
+done
+
+##########################################
+# Source machine runtime environment
+##########################################
+. ${HOMEgfs}/env/${machine}.env earc
+status=$?
+[[ ${status} -ne 0 ]] && exit $status
+
+##############################################
+# Obtain unique process id (pid) and make temp directory
+##############################################
+export pid=${pid:-$$}
+export DATA=${DATA:-${DATAROOT}/${jobid:?}}
+mkdir -p $DATA
+cd $DATA
+
+
+##############################################
+# Run setpdy and initialize PDY variables
+##############################################
+export cycle="t${cyc}z"
+setpdy.sh
+. ./PDY
+
+
+##############################################
+# Determine Job Output Name on System
+##############################################
+export pgmout="OUTPUT.${pid}"
+export pgmerr=errfile
+
+
+##############################################
+# Set variables used in the script
+##############################################
+export CDATE=${CDATE:-${PDY}${cyc}}
+export CDUMP=${CDUMP:-${RUN:-"gfs"}}
+export COMPONENT=${COMPONENT:-atmos}
+export n=$((ENSGRP))
```

Comment:
In `develop`, this is: 
```suggestion
export n=$((10#${ENSGRP}))
```

Reply:
Please move this line into `exgdas_enkf_earc.sh` and drop the `export`.

Reply:
This variable is used by hpssarch_gen.sh. Therefore will keep the export.

Reply:
Ok. 
From what I can tell, the variable in `hpssarch_gen.sh` is initializing from `n=1` and not really using this variable.
Anyways, you can leave it for another time.